.github/workflows/imports.yml

```diff
@@ -6,7 +6,7 @@ on:
 jobs:
   imports:
     runs-on: ubuntu-latest
-    container: obolibrary/odkfull:v1.4
```

Comment:
No you will definitely need that line, else you dont have the ODK installed you need to run the pipeline!